features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Documents/EmptyPage.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import Card from '@material-ui/core/Card';
+import Button from '@material-ui/core/Button';
+import PropTypes from 'prop-types';
+import {withStyles} from '@material-ui/core/styles';
+import { FormattedMessage, injectIntl } from 'react-intl';
+
+const styles = theme => ({
+    grid:{
+        marginTop:theme.spacing.unit*10
+    },
+    button:{
+        margin:theme.spacing.unit*2
+    },
+    typography:{
+        margin:theme.spacing.unit*2
+    }
+});
+
+function EmptyPage(props) {
+        const { classes } = props;
+        return(
+            <div>
```

Comment:
Shall we use [React Fragment](https://reactjs.org/docs/fragments.html) instead of `<div>` element to wrap the content of the component?